CONTRIBUTING.md

```diff
@@ -127,6 +127,10 @@ That will use the version of `web-ext` installed by this project.
 As with the build instructions above, after running `yarn build:safari` you
 should be able to run using Xcode.
 
+Note that Xcode will default to signing with Brian Birtles' ([@birtles](https://github.com/birtles)) team. You may need to override the team name to your personal team—however, please don't commit the configuration files this will change to this repo. This is clunky, but unavoidable for open source projects with Xcode.
```

Comment:
Do you remember having to sign up for an Apple Developer Program account for this? It's been such a long time since I was in this situation that I don't recall, but it would be nice to clarify for new contributors whether a personal team requires any extra work.

Link to enroll in the Apple Developer Program, in case it's relevant:
https://developer.apple.com/programs/enroll/

Reply:
I already had a Apple Developer Program account beforehand, so I'm not actually sure.

At least for building macOS binaries, I seem to recall that a Developer Program enrollment is required for notarizing, but not for just signing? Not sure if that holds for Safari plugins, though.